### PR TITLE
Update to Angular 8, Fix ViewChild

### PR DESCRIPTION
**Pull request by marlongehringer**
Updated Angular to Support Angular 8.
Fixed breaking change in ViewChild.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,47 +11,46 @@
...
     "intersection-observer": "^0.5.1",
-    "rxjs": "~6.3.3",
-    "tslib": "^1.9.0",
-    "zone.js": "~0.8.26"
+    "rxjs": "~6.5.2",
+    "tslib": "^1.10.0",
+    "zone.js": "~0.9.1"
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "~0.11.0",
```
Comment (mralexandernickel) [inline package.json]: Seems like this package also needs an update, see here:

https://github.com/angular/angular-cli/issues/15063

**Comment by marlongehringer** [comment]
True, had the same problem in another project. Should be fixed now.

**Comment by mralexandernickel** [comment]
Mh....Tests are failing now (1 test)...looks like the ViewChild (that adapted the new syntax) is undefined now 🤔

Do get some kind of notification about the build status of the PR?

And by the way: big THX for your effort here! 😘

**Comment by mralexandernickel** [comment]
Weird...thought that angular just changed this syntax with v8, but doesn't look like:

https://github.com/angular/angular/blob/8.2.5/packages/core/src/metadata/directives.ts#L197-L220

**Comment by marlongehringer** [comment]
Guess you need to set static to true because you are using it in ngOnInit.

**Comment by mralexandernickel** [comment]
> Guess you need to set static to true because you are using it in ngOnInit.

Exactly what i found out right now ^^

**Comment by mralexandernickel** [comment]
Ehhmmm...wait...static IS true 🤔

**Comment by mralexandernickel** [comment]
Ok found it...we exactly have the breaking change here. because we are using `*ngIf` and the templateReference combined, we are forced to use `AfterViewInit`. Will make a last local test with it

**Comment by mralexandernickel** [comment]
Damn. The test is just successful, because the chrome-headless on travis is slow enough 😅

**Comment by mralexandernickel** [comment]
Released as 0.8.0 @marlongehringer :)

**Pull request merged**